mac/Keyman4MacIM/Keyman4MacIM/KMAboutWindow/KMAboutWindowController.m

```diff
@@ -38,7 +38,9 @@ - (void)windowDidLoad {
 }
 
 - (IBAction)configAction:(id)sender {
-    [self.AppDelegate.inputController showPreferences:sender];
+    NSLog(@"Calling showPreferences from About window (i.e., the old way)...");
```

Comment:
I forgot to change this log statement now that we're not calling `showPreferences:`